src/docs/sphinx/theory_reference/solid.rst

```diff
@@ -0,0 +1,100 @@
+.. ## Copyright (c) 2019-2021, Lawrence Livermore National Security, LLC and
+.. ## other Serac Project Developers. See the top-level COPYRIGHT file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+===============
+Solid Mechanics
+===============
+
+Consider the kinematics of finite deformation
+
+.. raw:: html
+
+    <a title="Displacement_of_a_continuum.png: Sanpaz
+    derivative work: Nicoguaro, CC BY-SA 3.0 &lt;https://creativecommons.org/licenses/by-sa/3.0&gt;, via Wikimedia Commons" href="https://commons.wikimedia.org/wiki/File:Displacement_of_a_continuum.svg"><img  alt="Displacement of a continuum" src="https://upload.wikimedia.org/wikipedia/commons/thumb/8/8b/Displacement_of_a_continuum.svg/512px-Displacement_of_a_continuum.svg.png"></a>
```

Comment:
Feel free to ignore this, but here's a continuum potato figure I made back in grad school: 

![ContinuumPotatoConfigurations](https://user-images.githubusercontent.com/16393433/107788159-f0ab7f80-6d04-11eb-9ef5-bff0c72fc7ca.png)

It matches your notation slightly better (using Omega for domain rather than kappa, doesn't introduce a bunch of unused symbols) and we own the rights to it, so it'd be okay to host locally.

Here's a pdf version: [ContinuumPotatoConfigurations.pdf](https://github.com/LLNL/serac/files/5972741/ContinuumPotatoConfigurations.pdf)

Reply:
Perfect! Thanks! I was being lazy and used the most reasonable one from wikimedia commons. 😆

Reply:
oops, it looks like the one I gave you has a white background, and our sphinx theme is slightly off-white. Here's one with a transparent background (I think) that might look better on the page
![continuum_potato](https://user-images.githubusercontent.com/61714427/107803993-39206880-6d18-11eb-97c5-d19e2fcfa4fc.png)